File: Pcbc.Sermons/ClientApp/src/app/view-models/sermon-series-view-model.ts
import { AudioFileViewModel } from "./audio-file-view-model";

export class SermonSeriesViewModel {
  public sermonAudioFiles: AudioFileViewModel[];
  public name: string;
  public bannerNetworkPath: string;

  constructor() {
    this.sermonAudioFiles = [];
    this.name = "";
    this.bannerNetworkPath = "";
  }
}

File: Pcbc.Domain/ViewModels/SermonSeriesViewModel.cs
namespace Pcbc.Domain.ViewModels
{
    /// <summary>
    ///     Represents a sermon series which is a collection of sermons
    /// </summary>
    public class SermonSeriesViewModel
    {
        /// <summary>
        ///     The sermon files that can be played
        /// </summary>
        public List<AudioFileViewModel> SermonAudioFiles { get; set; }

        /// <summary>
        ///     name of the sermon series
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        ///     Path to the banner image for the series
        /// </summary>
        public string BannerNetworkPath { get; set; }
    }
}
